index.html

```diff
@@ -4124,6 +4135,60 @@ <h2><span class="chunk">cLLi</span> Content Light Level Information</h2>
         </section>
       </section>
 
+      <section id="dWLm-chunk">
+        <h2><span class="chunk">dWLm</span> Diffuse white luminance metadata</h2>
+        <p>The two-byte chunk type field contains the hexadecimal values</p>
+
+        <pre>
+<!-- xx xx xx xx
```

Comment:
```suggestion
64 57 4C 6D
```

Reply:
Suggest renaming dWLm chunk to conform with HEIF effort which labels it ndwt (Nominal Diffuse White)

Reply:
I support this idea: nDWt.